lib/sprinkle_dns/providers/route53_client.rb

```diff
@@ -51,12 +52,25 @@ def fetch_hosted_zones(filter: [])
         end
       end
 
-      if hosted_zones.size != filter.size
-        missing_hosted_zones = (filter - hosted_zones.map(&:name)).join(',')
-        raise MissingHostedZones, "Whooops, the following hosted zones does not exist: #{missing_hosted_zones}"
+      hosted_zones
+    end
+
+    def create_hosted_zones(hosted_zones)
+      change_requests = []
+
+      hosted_zones.each do |hosted_zone|
+        change_request = @api_client.create_hosted_zone({
+          name: hosted_zone.name,
+          caller_reference: "#{hosted_zone.name}.#{Time.now.to_i}",
```

Comment:
This show that it would be nice to have "run identifier", some identifier (like `uuid`), generated in the beginning of the whole run of the script.

Reply:
I like this idea, however, I think the `caller_reference` should be unique per "change_request", but I guess it could be `#{uuid}.#{Time.now.to_i}`, but currently we don't use  `caller_reference` for anything.